apps/web/screens/AboutPage/AboutPage.tsx

```diff
@@ -173,9 +210,9 @@ const Section: FC<SectionProps> = ({ slice, page, currentSliceId, setRef }) => {
                       </Typography>
                     </Stack>
                   </div>
-                  {slice.slices.map((slice) => (
+                  {(slice.slices as AvailableSlices[]).map((slice) => (
                     <Section
-                      key={slice.id}
+                      key={slice.__typename}
```

Comment:
any reason to not use slice.id?
using __typename would result in something weird happening when there's more than one slice of the same type

Reply:
Good catch, I change that I did and forgot to revert afterwards